update.sh

```diff
@@ -12,17 +12,20 @@ versions=( "${versions[@]%/}" )
 travisEnv=
 for version in "${versions[@]}"; do
 	travisEnv='\n  - VERSION='"$version$travisEnv"
-	
-	repoBase="http://packages.elasticsearch.org/elasticsearch/$version/debian"
-	if [ "${version%%.*}" -ge 2 ]; then
-		repoBase="http://packages.elasticsearch.org/elasticsearch/${version%%.*}.x/debian"
-	fi
-	
+
+	majorVersion="${version%%.*}"
+	repoBase="http://packages.elasticsearch.org/elasticsearch/${majorVersion}.x/debian"
+	case "$majorVersion" in
+		5) repoBase="https://artifacts.elastic.co/packages/${majorVersion}.x-prerelease/apt" ;;
```

Comment:
Since this is `-prerelease`, I wonder if we should rename the `5.0` folder to `5.0-rc` like we do in other repos (such as `docker`) to explicitly denote that it's tracking the pre-releases instead of just doing version-based hacking like this.  Thoughts?